### PR TITLE
fix(billing-cost-management-mcp-server): correct ECS and Lambda Compute Optimizer field names

### PR DESCRIPTION
**Pull request by pputra**
## Summary

  ### Changes

  Corrects AWS Compute Optimizer response field names for the ECS and Lambda
  recommendation tools in the billing-cost-management-mcp-server, and brings the
  `compute_optimizer_tools.py` test coverage to 100% (line and branch). Every
  field name was verified against the botocore `compute-optimizer/2019-11-01`
  service model and the public boto3 documentation.

  **ECS** (`get_ecs_service_recommendations`) — read three fields that do not
  exist in the API and therefore always returned `null`:
  - `currentPerformance` → **`currentPerformanceRisk`** (a string, not an object)
  - `currentServiceConfiguration.autoScalingGroupArn` → **`autoScalingConfiguration`**
    (a string enum; `autoScalingGroupArn` is the EC2 Auto Scaling shape)
  - option `projectedPerformance` → **`projectedUtilizationMetrics`** (an array of
    `{name, statistic, lowerBoundValue, upperBoundValue}`)

  **Lambda** (`get_lambda_function_recommendations`) — read `functionName`, which
  does not exist on `LambdaFunctionRecommendation`. The function name is now
  derived from `functionArn` (handling a trailing `:version`/`:alias`), matching
  how the RDS tool derives its instance name from the ARN.

  **Tests** — added/corrected unit tests so `compute_optimizer_tools.py` reaches
  100% line and branch coverage: ECS/Lambda assertions on the corrected fields, an
  ARN-derivation edge case, RDS storage options, EBS option parsing, ASG/EBS
  filter handling, per-operation dispatch, and the no-`max_results` path.

  ### User experience

  **Before:** ECS recommendations returned `null` for projected utilization,
  performance risk, and auto scaling configuration, and Lambda recommendations
  returned `null` for the function name — even when the API provided this data.
  For ECS this also caused the tool to appear as though the API "did not return"
  projected metrics.

  **After:** ECS recommendations return the real projected utilization metrics,
  performance risk, and auto scaling configuration, and Lambda recommendations
  return the function name (derived from the ARN). No output is dropped when the
  API provides it.

  ## Checklist

  If your change doesn't seem to apply, please leave them unchecked.

  * [x] I have reviewed the [contributing guidelines](https://github.com/awslabs/mcp/blob/main/CONTRIBUTING.md)
  * [x] I have performed a self-review of this change
  * [x] Changes have been tested
  * [x] Changes are documented

  Is this a breaking change? (Y/N) N

  **RFC issue number**: N/A

  Checklist:

  * [ ] Migration process documented
  * [ ] Implement warnings (if it can live side by side)

  ## Acknowledgment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the [project license](https://github.com/awslabs/mcp/blob/main/LICENSE).

### CODE REVIEW

**Review by wangyuhere — APPROVED** [review]

**Review by mattkelsey — APPROVED** [review]

**Review by Comusus — APPROVED** [review]

File: src/billing-cost-management-mcp-server/CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Corrected AWS Compute Optimizer recommendation response field names so EC2, Auto Scaling group, Lambda, and RDS tools return actual values instead of null. Fixed the shared savings-opportunity parser (`savingsOpportunityPercentage`), projected utilization metrics, EC2/RDS idle flags, nested ASG instance types, and the RDS instance/storage recommendation schema.
+- Corrected AWS Compute Optimizer ECS and Lambda recommendation response field names that read non-existent SDK fields and returned null. ECS now reads `currentPerformanceRisk`, `autoScalingConfiguration`, and `projectedUtilizationMetrics` (previously the non-existent `currentPerformance`, `autoScalingGroupArn`, and `projectedPerformance`), and Lambda derives the function name from the ARN (there is no `functionName` field).
```
Comment (scottschreckengaust) [inline src/billing-cost-management-mcp-server/CHANGELOG.md:18]: Waiting till GitHub Jobs are run
